lib/qunit-dom.ts

```diff
@@ -19,7 +19,7 @@ QUnit.assert.dom = function (
   rootElement = rootElement || this.dom.rootElement || document;
 
   if (arguments.length === 0) {
-    target = rootElement;
+    target = rootElement instanceof Element ? rootElement : null;
```

Comment:
This is basically the same check that `findElement`/`findElements` does internally, and will ensure that `ember-qunit` users are unaffected by these changes (since Ember users have a customized `QUnit.assert.dom.rootElement` that is the `#ember-testing` div).